### ISSUE TITLE
Found a Virus when i downloaded the repo

### ISSUE THREAD

**Issue by sohamxz** [Enigmatic-Mind/Monroy_Template_Vault#2]
Yeah So i am not sure but i found a "Wacatac.B!ml Trojan" The moment it infects your system, it puts you at risk for identity theft, data infection, and financial loss. Further, it will drain m
![GG2tsL4awAA3Vge](https://github.com/Enigmatic-Mind/Monroy_Template_Vault/assets/60619611/88265765-bdaf-4ec6-bf5f-fc37e88cf005)
any resources in the background without your knowledge

**Comment by Enigmatic-Mind** [comment]
@sohamxz what antivirus software did you use? I'll try replicating it on my windows computer to see which community plug-in its coming from or if its a false alarm.

**Comment by Enigmatic-Mind** [comment]
@sohamxz windows defender isn't finding anything; I have the vault zipped and unzipped as you can see on the left. Can you tell me more about which anti-virus software you are using, and what setting you have set for the scan if possible?
![threat scan](https://github.com/Enigmatic-Mind/Monroy_Template_Vault/assets/70162450/7bbcf585-bdaf-4e2b-91fc-dcf6a3fc2132)

I am also currently running sophos home premium to see if it detects anything.

**Comment by Enigmatic-Mind** [comment]
I rescanned with windows defender (just the downloads folder) and with the Sophos Home Premium antivirus (my university gives free access), but nothing was found:
![threat scan 3](https://github.com/Enigmatic-Mind/Monroy_Template_Vault/assets/70162450/1ec948c2-1b24-4807-9fba-b16c119c17cc)

**Comment by sohamxz** [comment]
Yeah So i have Malwarebytes premium and windows defender , So when ever i am trying to download the vault from GitHub as a zip file , I tried it like 5 times getting same Trojan 
![image](https://github.com/Enigmatic-Mind/Monroy_Template_Vault/assets/60619611/c29bd061-a1b6-4dfa-8371-9c3c53d5bc18)
![image](https://github.com/Enigmatic-Mind/Monroy_Template_Vault/assets/60619611/ea312f13-8175-4cf2-baf9-cda333a4006f)
So to check it wasn't the problem in my end or IDM or GitHub related i tried to downloaded "https://github.com/kepano/kepano-obsidian" which is kepanao's Obsidian Template Vault  
![image](https://github.com/Enigmatic-Mind/Monroy_Template_Vault/assets/60619611/4e86833a-8d77-44d0-baa6-84d1bae9564f)
which got downloaded without a problem or any Virus | Malware | Trojan 
![image](https://github.com/Enigmatic-Mind/Monroy_Template_Vault/assets/60619611/fc8da01e-846a-4c5c-94d3-dc78506ddb14)
All these are failed IDM Download logs , Which got interrupted due to the Antivirus.
I didnt want to compromise the System So i didnt  "Allow" the file to open !, So i could not run it in VirusTotal for Scan.

**Comment by Enigmatic-Mind** [comment]
I downloaded malwarebytes premium (trial), and it did not detect anything. 

Here are a few scenarios I can think of:
- Is your windows up to date? This could be a false positive.
- The only code I introduced is in the scripts folder (script_one.js and script_two.js). I did use ChatGPT to help write these, so maybe windows defender is detecting these because they are scripts and written by an ML?
- One of the community plug-ins has a virus, which would be pretty serious and important to the obsidian community.

I can create a branch from the main without the script folder and see if that changes anything. Everything else is just md files, images, and community plug-ins.

Let me now how you want to proceed. Thanks for bring this up and helping debug this.

![threat scan 4](https://github.com/Enigmatic-Mind/Monroy_Template_Vault/assets/70162450/045d78e4-ae80-4759-8c0a-75b58d04ae80)
